### PR TITLE
Chrome Dev Tools -  First Exercise

### PR DESCRIPTION
**Pull request by lizraeli**

### CODE REVIEW

File: unit1-intro-to-programming-in-javascript/chrome-dev-tools/ex1.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+- author: Lev
+- language: JavaScript
+- topics: Chrome DevTools
+- difficulty: 
+- format: 
+
+### Question
+For this exercise, you will need to run the relevant html page on a Chrome browser. After entering an input, open `Chrome Developer Tools` on the tab `soures` and put a break point inside the function **func2**.  Then click on the button and follow the variable **followMe**. Note the last value of this variable before the function returns.
```
Comment (bensussman) [inline unit1-intro-to-programming-in-javascript/chrome-dev-tools/ex1.md]: typo: `sources`

Comment (bensussman) [inline unit1-intro-to-programming-in-javascript/chrome-dev-tools/ex1.md]: "Click on the button"? Which button?

File: unit1-intro-to-programming-in-javascript/chrome-dev-tools/ex1.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            followMe += num % 3;
+          }
+
+          return num;
+        }
+      }
+    </script>
+
+  </BODY>
+</HTML>
```
Comment (bensussman) [inline unit1-intro-to-programming-in-javascript/chrome-dev-tools/ex1.html]: 👍

**Comment by bensussman** [comment]
Great! 👍  I love the HTML, i think the instructions could use some improvements in clarity.

**Comment by bensussman** [comment]
👍  Can i merge?

**Comment by lizraeli** [comment]
Sure 👍

**Pull request merged**